webplugin/js/app/km-utils.js

```diff
@@ -114,7 +114,8 @@ KommunicateConstants = {
         IMAGE:"9",
         CARD_CAROUSEL:"10",
         GENERIC_BUTTONS:"11", // supports link button/submit buttons and suggested replies 
-        FORM:"12"
+        FORM:"12",
+        BUTTON:"13"
```

Comment:
GENERIC_BUTTONS_V2  would be more appropriate

Reply:
@Suraj-Kumar "BUTTON" is renamed to GENERIC_BUTTONS_V2